packages/@aws-cdk/aws-cognito-identitypool-alpha/README.md

```diff
@@ -23,8 +23,8 @@
 [Amazon Cognito Identity Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-identity.html) enable you to grant your users access to other AWS services.
 
 Identity Pools are one of the two main components of [Amazon Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html), which provides authentication, authorization, and
-user management for your web and mobile apps. Your users can sign in directly with a user name and password, or through
-a third party such as Facebook, Amazon, Google or Apple.
+user management for your web and mobile apps. Your users can sign in through a a trusted identity provider, like a user 
```

Comment:
i would like to keep in the description for third party if that makes sense just so that it is clear that we support this in the module.
eg. 
`trusted identity provider along with third party providers such as Facebook, Amazon, Google or Apple. `